RELEASE-NOTES.md

```diff
@@ -37,6 +37,35 @@
 * Adjust streamlit commands to PuPr syntax
 * Resolved `-a` option conflict in `snow snowpark procedure update` command by removing short version of `--replace-always` option (it was conflicting with short version of `--check-anaconda-for-pypi-deps`).
 
+
+# v1.2.1
+## Fixes and improvements
+* Fix homebrew installation
+
+
+# v1.2.0
+
+## Backward incompatibility
+* Removed `snow streamlit create` command. Streamlit can be deployd using `snow streamlit deploy`
+* Removed short option names in compute pool commands:
+  * `-n` for `--name`, name of compute pool
+  * `-d` for `--num`, number of pool's instances
+  * `-f` for `--family`, instance family
+* Renamed long options in Snowpark services commands:
+  * `--compute_pool` is now `--compute-pool`
+  * `--num_instances` is now `--num-instances`
+  * `--container_name` is now `--container-name`
+
+## New additions
+* `snow streamlit init` command that creates a new streamlit project.
+* `snow streamlit deploy` support pages and environment.yml files.
```

Comment:
Please check 2.0.0 description and remove duplicates

Reply:
Done